test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj

```diff
@@ -7,11 +7,20 @@
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
   </PropertyGroup>
 
+  <PropertyGroup>
+      <!--<VSTestCLIRunSettings>$(MSBuildProjectDirectory)\.runsettings</VSTestCLIRunSettings>-->
+      <!--<VSTestCLIRunSettings>.\.runsettings</VSTestCLIRunSettings>-->
+      <VSTestCLIRunSettings>RunConfiguration.TargetPlatform=x64</VSTestCLIRunSettings>
```

Comment:
It seems that we should common.props to make it work when running test manually.